dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/Index.cshtml

```diff
@@ -52,6 +52,17 @@
             </govuk-summary-list-row>
         </govuk-summary-list>
 
+        @if (Model.Trn is not null)
```

Comment:
The linked designs didn't actually show the TRN, so I'm not sure if this is correct